packages/design-system-react/src/components/avatar-base/AvatarBase.tsx

```diff
@@ -43,9 +40,11 @@ export const AvatarBase = React.forwardRef<HTMLDivElement, AvatarBaseProps>(
       <Component ref={ref} className={mergedClassName} style={style} {...props}>
         {children || (
           <Text
-            variant={AVATAR_BASE_SIZE_TO_TEXT_VARIANT_MAP[size]}
-            fontWeight={FontWeight.Bold}
+            variant={TextVariant.BodySm}
+            fontWeight={FontWeight.Medium}
+            color={TextColor.TextMuted}
             asChild
+            className="uppercase"
```

Comment:
Change the variant to *BodySm*, set the font weight to *medium*, the text color to *text-muted*, and default to *uppercase*.